### PR TITLE
docs: Move Box, Flex, and Grid to a new Layout Folder in Storybook

### PR DESCRIPTION
**Pull request by thunguyen19**
#### Components
image

#### Testing
image

## Summary

Resolves: #2038 

## Release Category
Components

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

### CODE REVIEW

File: modules/react/layout/stories/stories_FlexVisualTesting.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {ContentDirection} from '@workday/canvas-kit-react/common';
 import {StaticStates, ComponentStatesTable} from '@workday/canvas-kit-react/testing';
 
 export default withSnapshotsEnabled({
-  title: 'Testing/Containers/Layout/Flex',
+  title: 'Components/Layout/Flex',
```
[the remark this answers is not in the document]
Comment (thunguyen19) [reply modules/react/layout/stories/stories_FlexVisualTesting.tsx:10]: @jaclynjessup Okay. Do we still keep it under `Containers/Layout` then?

Comment (jaclynjessup) [inline modules/react/layout/stories/stories_FlexVisualTesting.tsx:10]:
```suggestion
  title: 'Testing/Layout/Flex',
```

Comment (thunguyen19) [reply modules/react/layout/stories/stories_FlexVisualTesting.tsx]: Updated

**Comment by thunguyen19** [comment]
@jaclynjessup Please review again. I missed the checklist in the original issue. Everything should be good now.

**Review by jaclynjessup — APPROVED** [review]

**Review by mannycarrera4 — APPROVED** [review]

**Pull request merged**